modules/lang-painless/src/main/java/org/elasticsearch/painless/Def.java

```diff
@@ -146,6 +146,14 @@ private ArrayLengthHelper() {}
     /** factory for arraylength MethodHandle (intrinsic) from Java 9 (pkg-private for tests) */
     static final MethodHandle JAVA9_ARRAY_LENGTH_MH_FACTORY;
 
+    public static final MethodHandle DEF_TO_B_BYTE_IMPLICIT_HANDLE;
```

Comment:
Maybe be more explicit in the name, use "BOXED" rather than "B". Another thought on the naming is maybe HANDLE is not needed (we don't have it on any of the other method handles here). So as an example:

```
DEF_TO_BOXED_BYTE_IMPLICIT_CAST
```